hypervisor/qemu/qemu.go

```diff
@@ -354,4 +373,68 @@ func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
 		"-fsdev", fmt.Sprintf("local,id=virtio9p,path=%s,security_model=none", ctx.ShareDir),
 		"-device", fmt.Sprintf("virtio-9p-pci,fsdev=virtio9p,mount_tag=%s", hypervisor.ShareDirTag),
 	)
+
+	//lazy add nic device
+	for i, info := range qc.interfaces {
+		params = append(params,
+		"-netdev", fmt.Sprintf("tap,fd=%d,id=%s", info.fd, info.deviceName),
```

Comment:
qemu is launched via daemon mode, the all the fds will be closed. how these code works?